### PR TITLE
Mks 93 additional task

### PR DESCRIPTION
**Pull request by TsisinskyjV**

### CODE REVIEW

File: Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs [hunk 1]
```diff
@@ -36,14 +41,14 @@ public IActionResult Get()
...
         // GET api/Students/1
         [HttpGet("{id}")]
-        public IActionResult Get(int id)
+        public IActionResult GetById(int id)
         {
             try
             {
                 Student student = new Student();
-                student = sqlcomm.Select<Student>(String.Format("SELECT * FROM Student WHERE Id={0}", id));
+                student = sqlcomm.Select<Student>(String.Format("SELECT * FROM Student WHERE Id = {0}", id));
```
Comment (Magfiozy) [inline Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs]: Move BL to services

Comment (TsisinskyjV) [reply Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs]: done

**Review by bvzayats — COMMENTED** [review]
Why do you have StudentsDB.bak  in this PR?

File: Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs [hunk 2]
```diff
@@ -59,7 +54,14 @@ public IActionResult Post([FromBody] Student value)
         {
```
Comment (bvzayats) [inline Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs:50]: Post and Put params should be DTO

File: Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs [hunk 3]
```diff
@@ -87,6 +94,8 @@ public IActionResult Delete(int id)
...
             try
             {
+                sqlcomm.Delete(id);
+
                 return Ok();
```
Comment (bvzayats) [inline Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs]: Delete should return NoContent

File: Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs [hunk 4]
```diff
@@ -59,7 +54,14 @@ public IActionResult Post([FromBody] Student value)
         {
             try
             {
-                return Ok();
+                if (ModelState.IsValid)
```
Comment (Magfiozy) [inline Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs]: Are you sure that you should throw an exception if ModelState isValid?

**Comment by TsisinskyjV** [comment]
> Why do you have StudentsDB.bak in this PR?

I filled DB with some fields.

File: Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs (hunk 3 above)
Comment (TsisinskyjV) [reply Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs]: done

File: Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs (hunk 2 above)
Comment (TsisinskyjV) [reply Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs:50]: done

File: Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs (hunk 4 above)
Comment (TsisinskyjV) [reply Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs]: I checked it in last commit

**Review by bvzayats — COMMENTED** [review]
Why StudentsDB.bak file exists in this PR?

**Comment by TsisinskyjV** [comment]
> Why StudentsDB.bak file exists in this PR?

I filled DB with some fields.

**Review by bvzayats — APPROVED** [review]

**Review by sergiynikon — APPROVED** [review]

**Pull request merged**